### PR TITLE
Move to next screen on Enter

### PR DESCRIPTION
**Pull request by benthecarman**

### CODE REVIEW

**Review by benalleng — APPROVED** [review]
Looks good and works for me!

**Comment by futurepaul** [comment]
hey, this works but the "proper" way to do this is to use a `<form>` element so I'm going to do some add-on commits to this, will involve some layout changes

File: frontend/src/routes/ConnectPeer.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import MutinyToaster from "@components/MutinyToaster";
 import { useQuery } from "@tanstack/react-query";
 import { getFirstNode, getHostname, toastAnything } from "@util/dumb";
 import takeN from "@util/takeN";
-import { useContext, useState } from "react";
+import React, { useContext, useState } from "react";
```
Comment (futurepaul) [inline frontend/src/routes/ConnectPeer.tsx:7]: react types are global I think you don't need to import React for them

**Review by futurepaul — CHANGES_REQUESTED** [review]

**Review by futurepaul — APPROVED** [review]
this looks good to me now... tested each screen as I went

**Pull request merged**